src/components/IntegrationsView/IntegrationCard.module.css

```diff
@@ -0,0 +1,16 @@
+.integrationCard {
+  padding: 1.5rem;
```

Comment:
card has padding props, I'm not sure how it converts to rem.

Reply:
![image](https://github.com/user-attachments/assets/59822157-8492-4711-9eb8-f5fad9c19ada)
It seems, that card doesn't have padding or even margin props

Reply:
really?... that's odd :/

Reply:
it could go on the flex?